lr_face/data_providers.py

```diff
@@ -41,10 +41,25 @@ class ImageWithIds:
 
 
 class ImageProvider(Protocol):
-    def __call__(self, resolution: Tuple[int, int], *args, **kwargs) -> ImageWithIds:
+    def __call__(self, resolution: Tuple[int, int], *args,
+                 **kwargs) -> ImageWithIds:
         pass
 
 
+@dataclass
+class Triplet:
```

Comment:
de andere dataklasses bevatten zelf een aantal lijsten. Kan je die refactoren zodat ze ook dit formaat aanhouden? (of daar een taakje voor maken)?

Dat was ooit logisch toen het nog niet allemaal lijsten waren...

Reply:
Lijkt me een heel goed idee ja. Ik had ook op de agenda voor morgenochtend een puntje staan over het refactoren van dit soort dingen in de repo en hoe jullie daar tegenover staan, dus goed om te weten.

Reply:
Ik maak hier voor nu even een apart taakje voor.